### DISCUSSION TITLE
[Question] - Reference for Spell Points

### DISCUSSION THREAD

**Discussion by notaWizard11** [misthero/dnd5e-spellpoints#116] [Q&A]
Forgive my ignorance, I'm brand new to the world of Foundry and macros/code in general,

Is there a way to reference spell points through a macro or roll? I want to have a potion that, when consumed, replaces some spell points for an actor. I've looked as best I can through the console and can't find a reference of how to call or modify spell points other than editing the spell point max in the character sheet itself.

I'm thinking something similar to these: https://github.com/foundryvtt/dnd5e/wiki/Roll-Formulas

Any guidance/advice on if this is the wrong approach would be most appreciated!

**Comment by misthero** [comment]
hello @mbrueckl, that depends on what you are doing and the version of the module you are using. the new version uses an Item to handle spellpoints, so you can refer to the item.uses

![immagine](https://github.com/misthero/dnd5e-spellpoints/assets/3662610/db12eee5-d2b5-45b2-8cd8-c90a60ceef8d)


the old version uses resources:

![immagine](https://github.com/misthero/dnd5e-spellpoints/assets/3662610/87150da6-b2e2-4d1d-8820-cecef3a4af10)

**Comment by notaWizard11** [comment]
Thanks for taking the time to reply! That's great, I'm using the item version, so I'll play around with that,
